java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -391,6 +402,24 @@ private string stubMember(Member m) {
   )
 }
 
+private string stubAnnotation(Annotation a) {
+  if exists(a.getAValue())
+  then result = a.toString() + "(" + concat(stubAnnotationValue(a.getAValue()), ",") + ")"
+  else result = a.toString()
+}
+
+private string stubAnnotiationSimpleValue(Expr value) {
```

Comment:
There seems to be a typo: <code>stubAnnot<del>i</del>ationSimpleValue</code>

Reply:
Fixed in 2fb30d8, thanks!